HW4_Mukhametshina/amino_acid_tools.py

```diff
@@ -0,0 +1,241 @@
+
+def molecular_weight(sequence):
+    """
+    Function calculates molecular weight of the amino acid chain
+        Parameters:
+            sequence (str): each letter refers to one-letter coded proteinogenic amino acids or "random"
+        Returns:
+            weight (float) Molecular weight of tge given amino acid chain in Da
+        """
+    if sequence != 'random':
+        molecular_weights = dict(zip(short_code, weight_amino))
+        weight = sum(molecular_weights.get(aa, 0) for aa in sequence)
+        return weight
+    else:
+        len = int(input("введите желаемую длину: "))
+        bases = list(amino_acid)
+        amino_sequencqe = ''.join(random.choice(bases) for i in range(len))
+        molecular_weights = dict(zip(short_code, weight_amino))
+        weight = sum(molecular_weights.get(aa, 0) for aa in amino_sequencqe)
+        return "рандомная последовательнсть", amino_sequencqe, weight
+
+def amino_to_rna(amino_sequence):
+    """
+    Function translates an amino acid sequence into a possible RNA sequence
+        Parameters:
+                amino_sequence (str) or "random"
+        Returns:
+                (str) possible RNA sequence
+    """
+    if amino_sequence != 'random':
+        rna_sequence = ""
+
+        for aminoacid in amino_sequence:
+            if aminoacid in codon_table:
+                codons = codon_table[aminoacid]
+                # Selecting one random codon
+                codon = random.choice(codons)
+                rna_sequence += codon
+            else:
+                print("Unknown amino acid code: ", aminoacid)
+
+        return rna_sequence
+    else:
+        len = int(input("введите желаемую длину: "))
+        bases = list(amino_acid)
+        amino_sequencqe = ''.join(random.choice(bases) for i in range(len))
+        rna_sequence = ""
+
+        for aminoacid in amino_sequencqe:
+            if aminoacid in codon_table:
+                codons = codon_table[aminoacid]
+                # Selecting one random codon
+                codon = random.choice(codons)
+                rna_sequence += codon
+            else:
+                print("Unknown amino acid code: ", aminoacid)
+        return "рандомная последовательнсть", amino_sequencqe, rna_sequence
+
+
+def amino_seq_charge(amino_sequence):
+    """
+    Function evaluates the overall charge of the aminoacid chain in neutral aqueous solution (pH = 7)
+        Parameters:
+            amino_sequence (str): amino acid sequence of proteinogenic amino acids or "random"
+        Returns:
+            (str): "positive", "negative" or "neutral"
+    """
+    if amino_sequence != 'random':
+        aminoacid_charge = {'R': 1, 'D': -1, 'E': -1, 'K': 1, 'O': 1}
+        charge = 0
+        for aminoacid in amino_sequence.upper():
+            if aminoacid in 'RDEKO':
+                charge += aminoacid_charge[aminoacid]
+        if charge > 0:
+            return 'positiv'
```

Comment:
```suggestion
            return 'positive'
```
))

А вообще было бы здорово прямо заряд сам и вернуть наверное